lib/facter/pe_status_check.rb

```diff
@@ -219,6 +219,19 @@
     end
   end
 
+  chunk(:S0029) do
+    next unless PEStatusCheck.replica? || PEStatusCheck.postgres? || PEStatusCheck.primary?
+    # check if concurrnet connections to Postgres approaching 90% defined
+
+    maximum = PEStatusCheck.max_connections.to_i
+    current = PEStatusCheck.cur_connections.to_i
+    percent_used = (current / maximum) * 100
+    { S0029: percent_used >= 90 }
```

Comment:
you have the logic backwards here you want it to be true when we are less than 90% utilised
```suggestion
    { S0029: percent_used <= 90 }
```